error.c

```diff
@@ -3188,36 +3188,41 @@ rb_syserr_new_path_in(const char *func_name, int n, VALUE path)
 }
 #endif
 
+NORETURN(static void rb_mod_sys_tmp(VALUE exc, VALUE mod));
+
+static void
+rb_mod_sys_tmp(VALUE exc, VALUE mod)
```

Comment:
What does `tmp` mean?

Reply:
It meant a template for these codes

```c
    rb_extend_object(exc, mod);
    rb_exc_raise(exc);
```

But, I think `rb_mod_sys_fail_tmp` is better now.

Reply:
"Template" for what?

`sys_fail` comes from "system call failure", I think.
OTOH, this function itself is no longer related with system calls, as the exception made from `errno` is given as an argument, therefore I don't think `sys` fits it.
Since the role of this function is to "raise the given exception extending the given module", a name I though out was `rb_mod_exc_raise` (`mod` may not sound nice but could be consistent with other `rb_mod_*_fail` functions).

Reply:
@nobu 
Thanks for your comment.
I cloud understanded these functions role.

I intended to mean template  means "cut out common code to some function", but  it's badly in this case.

`rb_mod_exc_raise` is good, I think.  I'll rename this function.